src/disableButtons.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var designModeIsOn = document.designMode === 'on';
```

Comment:
gotta polyfill this just like the other file

Reply:
wap wap wahhh